### PR TITLE
Fix/styles

### PR DESCRIPTION
**Pull request by VL4DYSL4V**

### CODE REVIEW

File: slidez-core/src/main/java/com/binarystudio/academy/slidez/infrastructure/security/SecurityConfig.java [hunk 1]
```diff
@@ -14,10 +14,9 @@
 	private void applyRouteRestrictions(HttpSecurity http) throws Exception {
 		http.antMatcher("/**").authorizeRequests()
 				// PUBLIC
-				.antMatchers("/auth/**").permitAll()
-				.antMatchers("/swagger-ui/**").permitAll()
-				.antMatchers("/api-docs/**").permitAll()
-				.antMatchers(HttpMethod.GET, "/**").permitAll().anyRequest().authenticated();
+				.antMatchers("/auth/**").permitAll().antMatchers("/swagger-ui/**").permitAll()
+				.antMatchers("/api-docs/**").permitAll().antMatchers(HttpMethod.GET, "/**").permitAll().anyRequest()
+				.authenticated();
```
Comment (Kosichenko) [inline slidez-core/src/main/java/com/binarystudio/academy/slidez/infrastructure/security/SecurityConfig.java:19]: I think you shouln't change this.

**Review by Kosichenko — CHANGES_REQUESTED** [review]

File: slidez-fe-core/src/common/components/forms/sign-form.scss [hunk 2]
```diff
@@ -6,6 +6,7 @@
   flex-direction: column;
   width: 45%;
   margin-right: 10px;
+  font-family: 'Rubik', serif;
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/common/components/forms/sign-form.scss]: Font should be written in body, as global style, please, don't set it this way
If needed set it in global.scss to all div or what this component is)

File: slidez-fe-core/src/common/components/forms/sign-form.scss [hunk 3]
```diff
@@ -23,17 +24,15 @@
...
-.signUpLink, .link {
+.link {
   color: $main-color;
-  font-size: 14px;
   text-decoration: none;
-}
-.link {
-  font-size: 12px;
+  font-size: 14px;
+  font-family: 'Rubik', serif;
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/common/components/forms/sign-form.scss]: The same

File: slidez-fe-core/src/common/components/forms/sign-form.scss [hunk 4]
```diff
@@ -61,9 +60,10 @@
 }
 
 .label {
-  font-size: 12px;
+  font-size: 14px;
   margin-bottom: 5px;
   font-weight: 400;
+  font-family: 'Rubik', serif;
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/common/components/forms/sign-form.scss]: The same

File: slidez-fe-core/src/common/components/forms/sign-form.scss (hunk 2 above)
Comment (Anatolii-Ivanov) [reply slidez-fe-core/src/common/components/forms/sign-form.scss]: I mean font-family

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

File: slidez-core/src/main/java/com/binarystudio/academy/slidez/infrastructure/security/SecurityConfig.java (hunk 1 above)
Comment (VL4DYSL4V) [reply slidez-core/src/main/java/com/binarystudio/academy/slidez/infrastructure/security/SecurityConfig.java:19]: That was conflict that I resolved in favor of 'develop' branch

Comment (Kosichenko) [reply slidez-core/src/main/java/com/binarystudio/academy/slidez/infrastructure/security/SecurityConfig.java:19]: Compare it with dev branch. It is different.

**Review by Kosichenko — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**